CDCgov/NEDSS-Modernization
# Cnft1 1490 system management permissions

Added the system management access hook to hide system management based on the user permission rules

https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT1/boards/112?selectedIssue=CNFT1-1490

## Steps to verify

1. Log in using MSA as user and create a new user with only "view only" permissions.
2. Login locally with that new user you just created
3. Observe how "system management" is hidden
4. Another way of observing the permissions is by looking at the "me" endpoint and seeing the list of permissions based on the user
